### ISSUE TITLE
A11y_AzureDataStudio_Summary_ScreenReader:NVDA does not announces "blank" for the empty header.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7621]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#DesktopApp;#NVDA;#JAWS;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
NVDA Version:2019.2

**Repro Steps:**

1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next button.
8.	Start the NVDA and Navigate to table. 
9.    Observe the NVDA does not announces "blank" for the empty column header.

**Actual:**
NVDA does not announces the blank for the empty header present in the table.

**Expected:**
NVDA should  announces the blank for the empty header present in the table.

**Same issue observed in JAWS.**

**User Impact:**
Screen reader user does not understood that the control is empty.

**Attachment for Reference:**
[7621_A11y_AzureDataStudio_Summary_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3712516/7621_A11y_AzureDataStudio_Summary_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Data Studio
Version: 1.15.0-insider (user setup)
The issue is resolved, Hence closed this bug.
![image](https://user-images.githubusercontent.com/55909580/72047752-3ebfa980-32e1-11ea-9ae0-2b470651dc47.png)
